sdk-base/api/metalava.txt

```diff
@@ -59,6 +59,7 @@ package com.mapbox.maps.dsl {
   public final class CameraOptionsKtxKt {
     method public static inline com.mapbox.maps.CameraOptions cameraOptions(kotlin.jvm.functions.Function1<? super com.mapbox.maps.CameraOptions.Builder,kotlin.Unit> block);
     method public static inline com.mapbox.maps.CameraOptions cameraOptions(com.mapbox.maps.CameraState cameraState, kotlin.jvm.functions.Function1<? super com.mapbox.maps.CameraOptions.Builder,kotlin.Unit> block);
+    method public static boolean isEmpty(com.mapbox.maps.CameraOptions);
```

Comment:
is this supposed to be a public API? or should it be an internal API only?

Reply:
will be removed per discussion with @yunikkk - I'll create follow-up ticket later